NorthstarDLL/plugins/plugins.cpp

```diff
@@ -207,6 +207,11 @@ bool PluginManager::LoadPlugins()
 	data.version = ns_version.c_str();
 	data.northstarModule = g_NorthstarModule;
 
+	if (strstr(GetCommandLineA(), "-noplugins") != NULL)
+	{
+		NS::log::PLUGINSYS->warn("-noplugins detected; skipping loading plugins");
+		return false;
+	}
```

Comment:
I assume there's a reason for this but why isn't the check at the beginning of the function? ^^

Reply:
look at the if cases below